data-plane/THIRD-PARTY.txt

```diff
@@ -36,12 +36,13 @@ Lists of 229 third-party dependencies.
      (Unknown license) dispatcher (dev.knative.eventing.kafka.broker:dispatcher:1.0-SNAPSHOT - no url defined)
      (Unknown license) dispatcher-vertx (dev.knative.eventing.kafka.broker:dispatcher-vertx:1.0-SNAPSHOT - no url defined)
      (Unknown license) receiver (dev.knative.eventing.kafka.broker:receiver:1.0-SNAPSHOT - no url defined)
-     (The Apache Software License, Version 2.0) CloudEvents - API (io.cloudevents:cloudevents-api:2.5.0 - https://cloudevents.github.io/sdk-java/cloudevents-api/)
-     (The Apache Software License, Version 2.0) CloudEvents - Core (io.cloudevents:cloudevents-core:2.5.0 - https://cloudevents.github.io/sdk-java/cloudevents-core/)
-     (The Apache Software License, Version 2.0) CloudEvents - Vert.x Http Binding (io.cloudevents:cloudevents-http-vertx:2.5.0 - https://cloudevents.github.io/sdk-java/cloudevents-http-vertx/)
-     (The Apache Software License, Version 2.0) CloudEvents - JSON Jackson (io.cloudevents:cloudevents-json-jackson:2.5.0 - https://cloudevents.github.io/sdk-java/cloudevents-json-jackson/)
-     (The Apache Software License, Version 2.0) CloudEvents - Kafka Binding (io.cloudevents:cloudevents-kafka:2.5.0 - https://cloudevents.github.io/sdk-java/cloudevents-kafka/)
-     (The Apache Software License, Version 2.0) cloudevents-sql (io.cloudevents:cloudevents-sql:2.5.0 - https://cloudevents.github.io/sdk-java/cloudevents-sql/)
+     (Unknown license) receiver-vertx (dev.knative.eventing.kafka.broker:receiver-vertx:1.0-SNAPSHOT - no url defined)
+     (The Apache Software License, Version 2.0) CloudEvents - API (io.cloudevents:cloudevents-api:2.3.0 - https://cloudevents.github.io/sdk-java/cloudevents-api/)
+     (The Apache Software License, Version 2.0) CloudEvents - Core (io.cloudevents:cloudevents-core:2.3.0 - https://cloudevents.github.io/sdk-java/cloudevents-core/)
+     (The Apache Software License, Version 2.0) CloudEvents - Vert.x Http Binding (io.cloudevents:cloudevents-http-vertx:2.3.0 - https://cloudevents.github.io/sdk-java/cloudevents-http-vertx/)
+     (The Apache Software License, Version 2.0) CloudEvents - JSON Jackson (io.cloudevents:cloudevents-json-jackson:2.3.0 - https://cloudevents.github.io/sdk-java/cloudevents-json-jackson/)
+     (The Apache Software License, Version 2.0) CloudEvents - Kafka Binding (io.cloudevents:cloudevents-kafka:2.3.0 - https://cloudevents.github.io/sdk-java/cloudevents-kafka/)
+     (The Apache Software License, Version 2.0) cloudevents-sql (io.cloudevents:cloudevents-sql:2.3.0 - https://cloudevents.github.io/sdk-java/cloudevents-sql/)
```

Comment:
Why this CE SDK version change?

Reply:
There was a merger conflict and the CE SDK versions were changed in #3183. 
from My end, I have just added
```
(Unknown license) receiver-vertx (dev.knative.eventing.kafka.broker:receiver-vertx:1.0-SNAPSHOT - no url defined)
```

While resolving the merge conflict all of these are showing, I don't know why.

Reply:
You might need to rebase the latest main branch

Reply:
Oh, wait. I mistakenly removed the upgrades version. while resolving conflict.

Reply:
Done.